chatbot_app/api/routers/message_router.py

```diff
@@ -1,3 +1,56 @@
-from fastapi import APIRouter
+import logging
+from fastapi import APIRouter, Query, HTTPException
+from chatbot_app.schemas.message_schema import TelegramMessage, MessageHistoryResponse
+from chatbot_app.startup import client
 
 message_router = APIRouter()
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s [%(levelname)s] %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+
+async def connect_client():
+    if not client.is_connected():
```

Comment:
lepiej tak bo to bardziej odporne sprawdzanie polaczenia 

import asyncio

async def connect_client(retries=3, delay=1):
    for _ in range(retries):
        if client.is_connected():
            return
        await client.connect()
        await asyncio.sleep(delay)
    if not client.is_connected():
        raise HTTPException(status_code=500, detail="Failed to connect to Telegram")